data/en/mbs_0255.json

```diff
@@ -0,0 +1,699 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "mbs",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
+    "view_submitted_response": {
+        "enabled": true,
+        "duration": 900
+    },
+    "sections": [{
+            "id": "section",
+            "groups": [{
+                "id": "group",
+                "title": "",
+                "blocks": [{
+                        "type": "Introduction",
+                        "id": "introduction",
+                        "primary_content": [{
+                            "type": "Basic",
+                            "id": "use-of-information",
+                            "content": [{
+                                "list": [
+                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
+                                    "You can provide informed estimates if actual figures aren’t available.",
+                                    "We will treat your data securely and confidentially."
+                                ]
+                            }]
+                        }],
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
+                            "content": [{
+                                "title": "",
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Turnover",
+                                    "content": [{
+                                            "description": "Include:",
+                                            "list": [
+                                                "income from sub-contracted activities",
+                                                "commission",
+                                                "sales of goods purchased for resale",
+                                                "transport, insurance and packaging charges",
+                                                "revenue earned from other parts of the business, not named at the top of this page (please supply at fair value)"
+                                            ]
+                                        },
+                                        {
+                                            "description": "Exclude:",
+                                            "list": [
+                                                "VAT",
+                                                "revenue earned from other parts of the business, not named at the top of this page (please supply at fair value)"
```

Comment:
This copy was incorrect on the slides but has now been updated to:

- VAT
- income from the sale of fixed capital assets
- grants and subsidies
- insurance claims
- interest received